cypress/e2e/everything.cy.js

```diff
@@ -102,6 +102,21 @@ describe('Creating a New Incident', () => {
       .should('contain.text', 'Was a feature flag toggled recently?')
       .should('contain.text', 'Has there been an infrastructure changed recently?')
   })
+
+  it('can select the status through the keyboard', () => {
+    getDataTest('summary__select__status')
+      .click()
```

Comment:
I couldn't do `.focus()` on the element and while `shift-tab` works to get there through the current focus mechanism I felt `click()` was fair enough during a test.

I'd be okay with making it a bit more brittle by saying that when I landed on the page I'll type `shift-tab` on the `what` field and then type in what I want. 

What do you think, @gabehollombe? :)

Reply:
If you want to test shift-tab specifically, go for it. I'm ok with either. Brittle is fine for now; throw a comment in saying 'if this breaks too often, change to click()'  =-)

Reply:
expecting `shift-tab` feels fair because it's about not changing expectations for the InCom and that's an important part of not regressing!

Reply:
It turns out Cypress doesn't handle tab keys out of the box. There is a plugin for it but let's just stick with tab for now.